src/Integration.Vsix/Resources/Strings.Designer.cs

```diff
@@ -500,7 +500,7 @@ public static string ToolsOptions_ErrorOpeningSettingsFile {
         ///      &quot;language&quot;: &quot;cpp&quot;
         ///    },
         ///    {
-        ///      &quot;language&quot; [rest of string was truncated]&quot;;.
+        ///      &quot;langu [rest of string was truncated]&quot;;.
```

Comment:
change by accident?

Reply:
Oh, yes. Thank you for the catch!

Reply:
Ah, no. That's the designer.cs, which is generated